lib/errors.py

```diff
@@ -0,0 +1,141 @@
+def handle_error(error_dict, status_code):
+    if status_code in error_dict.keys():
+        return error_dict[status_code]
+    elif status_code in errors.keys():
+        return errors[status_code]
+    else:
+        return "Unknown error"
+
+
+errors = {
+    400: "Failed due to malformed JSON.",
+    401: "Missing or invalid access token provided.",
+    403: "Missing or invalid access token provided.",
+    404: "A non-existent entity request.",
+    409: "Entity already exist.",
+    415: "Missing or invalid content type.",
+    422: "Database can't process request.",
+    500: "Unexpected server-side error occurred.",
+}
+users = {
+    "create": {
+        409: "Failed due to using an existing email address.",
+    },
+    "login": {
+        409: "Failed due to using an existing email address.",
+    },
+    "get": {
+        400: "Failed due to malformed query parameters.",
+    },
+    "get_all": {
+        400: "Failed due to malformed query parameters.",
+    },
+    "update": {
+        404: "Failed due to non existing user.",
+    },
+    "update_password": {
```

Comment:
Please double-check all the empty errors. Also, please check if there are multiple possible errors for different actions (e.g. `malformed` and `conflict`).